docs/getting-started.md

```diff
@@ -4,14 +4,18 @@ weight = 3
 nav = [
      "Starting Pilosa",
      "Sample Project",
+     "Using Curl",
+     "Using Go",
+     "Using Java",
+     "Using Python",
      "What's Next?",
 ]
 +++
 
 ## Getting Started
 
 Pilosa supports an HTTP interface which uses JSON by default.
-Any HTTP tool can be used to interact with the Pilosa server. The examples in this documentation will use [curl](https://curl.haxx.se/) which is available by default on many UNIX-like systems including Linux and MacOS. Windows users can download curl [here](https://curl.haxx.se/download.html).
+Any HTTP tool can be used to interact with the Pilosa server. The examples in this documentation will use curl which is available by default on many UNIX-like systems including Linux and MacOS. However, the best way to interface with the Pilosa server is through one of our three client libraries. Pilosa currently supports [Go](https://github.com/pilosa/go-pilosa), [Java](https://github.com/pilosa/java-pilosa), and [Python](https://github.com/pilosa/python-pilosa).
```

Comment:
"one of our three *official* client libraries" - we do have others, but we focus on these three because they're full-featured, and we keep them up to date.